src/addons/Responsive/Responsive.js

```diff
@@ -60,7 +60,7 @@ export default class Responsive extends Component {
   constructor(...args) {
     super(...args)
 
-    this.state = { width: isBrowser() ? window.innerWidth : 0 }
+    this.state = { width: isBrowser() ? document.documentElement.clientWidth : 0 }
```

Comment:
This is probably worth adding an inline comment with the GitHub issue URL and a one-line description as to why this is being used.

Reply:
Okay added a comment and reference to the PR. Will be squashed if it's fine :)